NOTICE

```diff
@@ -8,5 +8,5 @@ within the component is also subject to the terms and conditions
 of these licenses. The LICENSE file contains the text of all the
 licenses which apply within this component.
  
-Copyright 2017 Metrological
-Licensed under the Apache License, Version 2.0
+Some images come from https://apc01.safelinks.protection.outlook.com/?url=https%3A%2F%2Furldefense.com%2Fv3%2F__http%3A%2F%2Fwww.pexels.com__%3B!!CQl3mcHX2A!TGkHa6hBR_OZdRsNJX9SFZI3t8nxhGDiDdHmT6hKoKPLGdYBwzLERSCNi_cVCQkrVWQs%24&amp;data=02%7C01%7Csukanya.i%40tataelxsi.co.in%7C30108378981a40fe5a3e08d7f2990986%7Cad6a39dd96b6436882daf2ec4d92e26a%7C0%7C0%7C637244611423025772&amp;sdata=gJicuP1BddQbagrMR1vhsyC3Wm%2FwyHlS31wK%2FX0BHhY%3D&amp;reserved=0  and are licensed under the Pexels License at: https://apc01.safelinks.protection.outlook.com/?url=https%3A%2F%2Furldefense.com%2Fv3%2F__https%3A%2F%2Fwww.pexels.com%2Flicense%2F__%3B!!CQl3mcHX2A!TGkHa6hBR_OZdRsNJX9SFZI3t8nxhGDiDdHmT6hKoKPLGdYBwzLERSCNi_cVCXcBL3RO%24&amp;data=02%7C01%7Csukanya.i%40tataelxsi.co.in%7C30108378981a40fe5a3e08d7f2990986%7Cad6a39dd96b6436882daf2ec4d92e26a%7C0%7C0%7C637244611423025772&amp;sdata=vI9dAppn%2FGJqqbUTYFuqQTrx%2Bdvs7KjVi1e5CqYq0Qw%3D&amp;reserved=0
```

Comment:
Sorry, the Accenture mailer is eating the URLs into horrible things.  What the text should say is:
Some images come from https://www.pexels.com and are licensed under the Pexels License at: https://www.pexels.com/license/
Credits: Scott Webb, Pixabay